src/assets/scripts/client/tutorial/TutorialView.js

```diff
@@ -201,8 +201,8 @@ export default class TutorialView {
                 if (prop.aircraft.list.length <= 0) {
                     return t;
                 }
-
-                return t.replace('{CALLSIGN}', prop.aircraft.list[0].callsign);
+                const reverseList = [...prop.aircraft.list].reverse();
```

Comment:
Using the spread here actually gives us a new array, leaving the original unaffected by the following .reverse()